subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AndroidGradleRecipesKotlinSmokeTest.groovy

```diff
@@ -136,7 +136,9 @@ class AndroidGradleRecipesKotlinSmokeTest extends AbstractSmokeTest {
         result.task(":app:$taskName").outcome == TaskOutcome.SUCCESS
 
         and:
-        assertConfigurationCacheStateStored()
```

Comment:
❓ What's the deal with adding all of these is CC enabled checks?  Isn't it enabled by default now?

Reply:
We test with both CC enabled and not enabled. 
Previously the `assertConfigurationCacheStateStored` method did this check, but since different smoke tests have different ways of doing that check, we need to lift the check into the smoke test itself